src/Symfony/Component/Config/Resource/ReflectionClassResource.php

```diff
@@ -154,6 +154,13 @@ private function generateSignature(\ReflectionClass $class): iterable
         }
 
         foreach ($class->getMethods(\ReflectionMethod::IS_PUBLIC | \ReflectionMethod::IS_PROTECTED) as $m) {
+            foreach ($this->excludedVendors as $vendor) {
+                $file = $m->getFileName();
+                if (str_starts_with($file, $vendor) && false !== strpbrk(substr($file, \strlen($vendor), 1), '/'.\DIRECTORY_SEPARATOR)) {
```

Comment:
```suggestion
                if (str_starts_with($file, $vendor.'/') || str_starts_with($file, $vendor.\DIRECTORY_SEPARATOR)) {
```

Would that be significantly slower?

Reply:
I went another way that you might still prefer